src/screens/InspectionCapture/index.js

```diff
@@ -75,7 +75,7 @@ export default function InspectionCapture() {
       try {
         const promises = Object.values(mapTasksToSights)
           .filter(((taskBySight) => sightIds.includes(taskBySight.id)))
-          .map((taskBySight) => taskBySight.task.name)
+          .map((taskBySight) => taskBySight)
```

Comment:
You can remove this map() as it isn't doing anything and just returning the same value.